src/engine.go

```diff
@@ -162,18 +162,22 @@ func (e *engine) print() string {
 			break
 		}
 		// escape double quotes contained in the prompt
-		prompt := fmt.Sprintf("PS1=\"%s\"", strings.ReplaceAll(e.string(), "\"", "\"\""))
-		prompt += fmt.Sprintf("\nRPROMPT=\"%s\"", e.rprompt)
+		prompt := fmt.Sprintf("PS1=\"%s\"", strings.ReplaceAll(strings.ReplaceAll(e.string(), "\"", "\"\""), "`", "'"))
```

Comment:
I'd rather split this into multiple lines. Right now it's not really readable.

Reply:
no it's not... and I'm afraid more things will come